docs/guide.md

```diff
@@ -0,0 +1,210 @@
+# Getting Started with PatternPack
```

Comment:
It looks like making that change will require a rework of most of the document :(